### PR TITLE
Added: Create Capture and Refund operation method

### PR DESCRIPTION
**Pull request by pitiphong-p**

### CODE REVIEW

**Review by igroomgrim — APPROVED** [review]

File: OmiseSwift/Charge.swift [hunk 1]
```diff
@@ -204,24 +204,30 @@ extension Charge {
...
+    
+    public static func capture(using given: Client? = nil, id: String, callback: @escaping CaptureOperation.Callback) -> Request<CaptureOperation.Result>? {
+        let operation = captureOperation(id: id)
         let client = resolveClient(given: given)
         return client.call(operation, callback: callback)
     }
     
-    public static func reverse(using given: Client? = nil, id: String, callback: @escaping ReverseOperation.Callback) -> Request<ReverseOperation.Result>? {
-        let operation = ReverseOperation(
+    public static func reverse(id: String) -> CaptureOperation {
```
Comment (chakrit) [inline OmiseSwift/Charge.swift]: Wait. Reverse returns `CaptureOperation`?

**Review by chakrit — CHANGES_REQUESTED** [review]

File: OmiseSwift/Charge.swift (hunk 1 above)
Comment (pitiphong-p) [reply OmiseSwift/Charge.swift]: T^T

Comment (pitiphong-p) [reply OmiseSwift/Charge.swift]: Compiler didn't complain so I think everything is fine Orz

Comment (pitiphong-p) [reply OmiseSwift/Charge.swift]: Fixed

**Review by chakrit — APPROVED** [review]

**Pull request merged**